test/htmlwriter.jl

```diff
@@ -77,15 +77,17 @@ end
         link = assetlink("foo/bar.md", asset)
         @test link.href == "bar.css"
     end
-    @test_throws Exception asset("ftp://example.com/foo.js", islocal=false)
-    @test_throws Exception asset("example.com/foo.js", islocal=false)
+    @test_throws Exception asset("ftp://example.com/foo.js", islocal = false)
+    @test_throws Exception asset("example.com/foo.js", islocal = false)
     @test_throws Exception asset("foo.js", islocal = false)
     @test_throws Exception asset("https://example.com/foo.js?q=1")
-    @test_throws Exception asset("https://example.com/foo.js", class=:error)
+    @test_throws Exception asset("https://example.com/foo.js", class = :error)
+    @test_throws Exception asset("https://example.com/foo.js", dst = "extra_assets/foo.js")
+    @test_throws Exception asset("https://example.com/foo.js", islocal = true)
     # Edge cases that do not actually quite work correctly:
-    let asset = asset("https://example.com/foo.js", islocal = true)
+    let asset = asset("https://example.com/foo.js", islocal = false)
```

Comment:
Not quite sure how i should change this.
I added more checks which lead this to throw exceptions.
After changing this your tests of the links throw errors.
Shall i drop those checks?

Reply:
I deleted those lines.